modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/playback/SleepTimer.kt

```diff
@@ -32,6 +32,7 @@ class SleepTimer @Inject constructor(
         private const val NUMBER_OF_CHAPTERS_KEY = "number_of_chapters"
         private const val END_OF_EPISODE_VALUE = "end_of_episode"
         private const val END_OF_CHAPTER_VALUE = "end_of_chapter"
+        val TAG: String = SleepTimer::class.java.name
```

Comment:
You probably don't want to do it this way. R8 will obfuscate the class name.

Reply:
I've updated it!